jira_automation/src/main/java/io/dockstore/jira/Utils.java

```diff
@@ -1,23 +1,111 @@
 package io.dockstore.jira;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpRequest.Builder;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.util.Base64;
 import java.util.List;
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import org.kohsuke.github.GHIssue;
 import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHMilestone;
 import org.kohsuke.github.GHRepository;
 import org.kohsuke.github.GitHub;
 import org.kohsuke.github.GitHubBuilder;
+import org.kohsuke.github.PagedIterable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class Utils {
 
+    public static final String JIRA_REST_BASE_URL = "https://ucsc-cgl.atlassian.net/rest/api/3/";
+    public static final String GITHUB_OPEN_ENDED_RESEARCH_TASKS = "Open ended research tasks";
```

Comment:
Might be a good idea just to change the label name in github? (i.e. I don't think we gain any value with mismatched names)

Reply:
Updated it in GitHub, left message on dockstore-devs. Going to leave the code to be safe.